tests/hooks/test_audit_logger.py

```
@@ -0,0 +1,184 @@
+        git_info = {"repo": "test-repo", "branch": "main", "commit": "abc123", "dirty": True}
+        system_info = {"hostname": "test-host", "username": "test-user"}
+
+        entry = format_log_entry(sample_hook_input, git_info, system_info)
+
+        assert "[dirty]" in entry
+
+
+class TestMain:
+    """Tests for main function."""
+
+    def test_invalid_json_exits_with_code_1(self, temp_claude_dir):
```

Ruff (0.14.14)

[warning] Unused method argument: `temp_claude_dir`

(ARG002)